### PR TITLE
strictly select the 2nd (eq(1)) link in Facebook post HTML

### PR DESCRIPTION
**Pull request by mlsteele**
Client side of https://github.com/keybase/proofs/pull/55

### CODE REVIEW

File: pvl-tools/pvl.cson [hunk 1]
```diff
@@ -99,12 +99,11 @@ services:
 
     # This query operates on the newly extractaparsed comment, not the original page load
     # This is the selector for the post attachment links, which contains the
-    # proof text. It's the "<a> tags inside the div that's the immediate
-    # *sibling* of the 'userContet' div". The second of these three <a> tags
-    # contains the proof text, the others are blank. But we just check their concatenation.
+    # proof text. It's the second <a> tag inside the div that's the immediate
+    # *sibling* of the 'userContet' div. The second of these three <a> tags
+    # contains the proof text, the others are blank. We check only the second.
```
Comment (oconnor663) [inline pvl-tools/pvl.cson]: The "others are blank" comment is now out of date. Can we update it? In this particular case, I think there ends up being more than three <a> tags :p

**Comment by oconnor663** [comment]
LGTM apart from the comment change above. Were you able to test that this works with olegatro's account, or is that blocked on the server change going in?

**Comment by mlsteele** [comment]
Yep, tested and it works for `olegatro`. Not blocked because the server has a hint :)
```
HOME=/tmp/chk keybase --run-mode=prod --standalone id -s olegatro
```

**Comment by mlsteele** [comment]
Updated the comment

**Pull request merged**